### DISCUSSION TITLE
Implement multiple shadows

### DISCUSSION THREAD

**Discussion by zhouzq-thu** [marcomusy/vedo#562] [Ideas]
- Implement multiple shadows on planes, using the projection algorithm from #191.

![Screen Shot 2021-12-15 at 3 40 59 PM](https://user-images.githubusercontent.com/9458157/146147924-c904144f-2a27-4b44-a672-c83c834506ac.png)

![Screen Shot 2021-12-15 at 3 30 14 PM](https://user-images.githubusercontent.com/9458157/146147969-64cdad41-2142-434f-8a4f-110016092cae.png)

- But there are more to do, such as clip points which are inside the plane.

![Screen Shot 2021-12-15 at 4 17 25 PM](https://user-images.githubusercontent.com/9458157/146149740-51ea21df-ee68-4c6d-a31f-0f7be83db347.png)

**Comment by marcomusy** [comment]
nice! :)
